### PR TITLE
Feature/bbb 6 update user details

### PR DESCRIPTION
**Pull request by pauljroberts**

### CODE REVIEW

File: api-acceptance-tests/src/test/resources/features/users.delete.feature [hunk 1]
```diff
@@ -0,0 +1,17 @@
+@users
```
Comment (sampathmahavithana) [inline api-acceptance-tests/src/test/resources/features/users.delete.feature]: This tag should be changed to match with the functionality. Ex. @delete_users

File: api-acceptance-tests/src/test/resources/features/users.delete.feature [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+Feature: Verify users retrieval
```
Comment (sampathmahavithana) [inline api-acceptance-tests/src/test/resources/features/users.delete.feature]: This feature should be changed to match the features testing. Ex. Verify user deletion

File: api-acceptance-tests/src/test/resources/features/users.get.feature [hunk 3]
```diff
@@ -0,0 +1,53 @@
+@users
```
Comment (sampathmahavithana) [inline api-acceptance-tests/src/test/resources/features/users.get.feature]: Same as above

File: api-acceptance-tests/src/test/resources/features/users.get.feature [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+Feature: Verify users retrieval
```
Comment (sampathmahavithana) [inline api-acceptance-tests/src/test/resources/features/users.get.feature]: Same as above

File: api-acceptance-tests/src/test/resources/features/users.post.feature [hunk 5]
```diff
@@ -1,50 +1,8 @@
 @users
```
Comment (sampathmahavithana) [inline api-acceptance-tests/src/test/resources/features/users.post.feature]: Same as above

File: api-acceptance-tests/src/test/resources/features/users.put.feature [hunk 6]
```diff
@@ -0,0 +1,49 @@
+@users
```
Comment (sampathmahavithana) [inline api-acceptance-tests/src/test/resources/features/users.put.feature]: Same as above

File: api-acceptance-tests/src/test/resources/features/users.delete.feature (hunk 1 above)
Comment (pauljroberts) [reply api-acceptance-tests/src/test/resources/features/users.delete.feature]: OK.  I'll do it in the Delete story.

File: api-acceptance-tests/src/test/resources/features/users.put.feature (hunk 6 above)
Comment (pauljroberts) [reply api-acceptance-tests/src/test/resources/features/users.put.feature]: Will fix as part of delete story if ok with everyone.  Delete will be done today hopefully.

Comment (sampathmahavithana) [reply api-acceptance-tests/src/test/resources/features/users.put.feature]: Sure thing. I'm happy with that.

**Pull request merged**